extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,72 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+/**
+ * @returns {number} the correct optimizely id to activate depending on the current href
+ */
+ve.init.wikia.getToolbarABTestId = function () {
+	// Production tests are split by language. This is done because each language has a separate engagement percentage
+	var i, href = window.location.href,
+		data = [
+			{
+				regex: /http:\/\/sandbox-japan\.visualeditor\.wikia\.com/,
+				id: 4701112678
+			},
+			{
+				regex: /http:\/\/ja\.[a-z0-9-]+\.wikia\.com/,
+				id: 4701112678
+			},
+			{
+				regex: /http:\/\/yaruo\.wikia\.com/,
+				id: 4701112678
+			},
+			{
+				regex: /http:\/\/es\.[a-z0-9-]+\.wikia\.com/,
+				id: 5003533755
+			},
+			{
+				regex: /http:\/\/de\.[a-z0-9-]+\.wikia\.com/,
+				id: 5013830116
+			},
+			{
+				regex: /http:\/\/[a-z0-9-]+\.wikia\.com/,
+				id: 5003080344
+			}
+		];
+
+	if ( window.wgDevelEnvironment ) {
+		return 4721410313;
+	} else {
+		for ( i = 0; i < data.length; i++ ) {
+			if ( href.match( data[i].regex ) ) {
+				return data[i].id;
```

Comment:
If you want to make a decision bases on the language - use global variable ```wgContentLanguage``` or ```mw.config.get('ContentLanguage')```. If you want to make a decision based on the wikiname - use global variable ```wgDBname```.
Also - I think the point here is to activate one test? If that's correct then you can ```break;``` inside the ```if``` statement.